st2api/Makefile

```diff
@@ -59,7 +59,7 @@ all: install
 
 install: wheelhouse changelog
 
-post_install:
+post_install: bdist_wheel
```

Comment:
yeah you are right, it's dependent somehow, just  made `pip list` >>> wheel (0.24.0)
it's there